README.md

```diff
@@ -1,13 +1,11 @@
-BibleMultiConverter
-===================
+# BibleMultiConverter
```

Comment:
Why?

You may ask Why Not? 2-line headlines in markdown are a lot more visible when editing the file in an editor without syntax highlighting, and the syntax is equivelent in both Markdown and RestructuredText.